docs/14_publish.md

```diff
@@ -1,15 +1,35 @@
 # Publishing Dandisets
 
-Once you create a Dandiset, DANDI will automatically create a `draft` version of the Dandiset that 
-can be 
-changed as many times as needed. 
-
-Prior to publishing, you should edit your metadata appropriately
-(e.g. people and funding contributors, protocol information, keywords, related resources in other 
-sources such as publications, code repositories, and other data).
-If you need to change the Dandiset or its metadata in the future, you will need to create another version of your 
-Dandiset. The content of the specific published version of the Dandiset cannot be changed.
-
-When you are ready to publish your data and to create a unique version of your 
-Dandiset, click the `Publish` button (on the right panel of the landing page). Note that the `Publish` 
-button is only active if all the metadata and asset errors are fixed.
+Once you create a Dandiset, DANDI will automatically create a `draft` version
+of the Dandiset that can be changed as many times as needed by editing the 
+metadata or uploading new files.
+
+When the draft version is ready, you can *publish* your Dandiset. This results
+in an immutable snapshot of your Dandiset with its own unique version number
+that others can cite. If you need to change the data or metadata, you can do
+so by continuing to modify the draft version and publishing a new version
+when you are ready.
+
+Follow these steps to publish your Dandiset:
+
+1. Edit the Dandiset metadata, aiming to fix all Dandiset metadata validation
+   errors, and include any other useful information. For example, you may want
+   to edit the following fields:
+    - People and funding contributors
+    - Protocol information
+    - Keywords
+    - Related resources such as publications and code repositories
+
+1. Fix all asset metadata errors by modifying the asset files to eliminate
+   the errors and re-uploading them.
+
+1. When all the Dandiset metadata and asset metadata errors are fixed, the
```

Comment:
```suggestion
1. When all the Dandiset metadata and asset metadata errors are fixed, and the dandiset made public if it was initially embargoed, the
```

I think worth mentioning, but we might need a dedicated doc to describe how.

Reply:
ping on this one @kabilar so we could potentially quickly merge this one

Reply:
Thanks @yarikoptic.  Just added this suggestion.